dss_datamover/utils/config.py

```diff
@@ -255,6 +261,12 @@ def delete(self, subparser):
                                help='Run DataMover in debug mode')
         subparser.add_argument("--profile", "-pro", required=False, action='store_true',
                                help='Profiling of DEL operation (Not Implemented)')
+        subparser.add_argument("--nfs-server", "-ns", type=str, required='--nfs-share' in sys.argv,
```

Comment:
This required field doesn't work for all modes except command line arguments on console. It is ok for now